src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "../utils/variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Arial", sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  width: 727px;
+  padding: 10px;
+  margin: 0 auto;
+  gap: $gap-month;
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} {
+      .calendar__item:nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  @each $item, $number in $days {
+    &--start-day--#{$item} > :first-child {
+      margin-left: calc((#{$cell-size + $gap-month}) * #{$number});
+    }
+  }
+
+  &__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+    list-style: none;
+    transition-duration: 0.5s;
+
+    @for $i from 1 through 31 {
```

Comment:
It is better to set cycles below the usual stylization.

Reply:
I wanted to show which cycle to which tag